### DISCUSSION TITLE
JavaFX NetBeans Issues

### DISCUSSION THREAD

**Discussion by SHISIA** [apache/netbeans#3664] [General]
### Apache NetBeans version

Apache NetBeans 12.6

### What happened

Slow and hanging. The IDE could not read VM options in JAVAFX. The image paths could not be displayed no matter how many times I modified the paths. There is seriously something wrong with netbeans.

-------------------------------------------------------------------------------
>Log Session: Wednesday, February 23, 2022 at 7:19:36 AM East Africa Time
>System Info: 
  Product Version         = Apache NetBeans IDE 12.6
  Operating System        = Windows 10 version 10.0 running on amd64
  Java; VM; Vendor        = 15.0.1; Java HotSpot(TM) 64-Bit Server VM 15.0.1+9-18; Oracle Corporation
  Runtime                 = Java(TM) SE Runtime Environment 15.0.1+9-18
  Java Home               = C:\Program Files\Java\jdk-15.0.1
  System Locale; Encoding = en_US (nb); Cp1252
  Home Directory          = C:\Users\SHISIA
  Current Directory       = C:\Program Files\NetBeans-12.6
  User Directory          = C:\Users\SHISIA\AppData\Roaming\NetBeans\12.6
  Cache Directory         = C:\Users\SHISIA\AppData\Local\NetBeans\Cache\12.6
  Installation            = C:\Program Files\NetBeans-12.6\netbeans\nb
                            C:\Program Files\NetBeans-12.6\netbeans\ergonomics
                            C:\Program Files\NetBeans-12.6\netbeans\ide
                            C:\Program Files\NetBeans-12.6\netbeans\extide
                            C:\Program Files\NetBeans-12.6\netbeans\java
                            C:\Program Files\NetBeans-12.6\netbeans\apisupport
                            C:\Program Files\NetBeans-12.6\netbeans\webcommon
                            C:\Program Files\NetBeans-12.6\netbeans\websvccommon
                            C:\Program Files\NetBeans-12.6\netbeans\enterprise
                            C:\Program Files\NetBeans-12.6\netbeans\profiler
                            C:\Program Files\NetBeans-12.6\netbeans\php
                            C:\Program Files\NetBeans-12.6\netbeans\harness
                            C:\Program Files\NetBeans-12.6\netbeans\cpplite
                            C:\Program Files\NetBeans-12.6\netbeans\groovy
                            C:\Program Files\NetBeans-12.6\netbeans\javafx
                            C:\Program Files\NetBeans-12.6\netbeans\platform
  Boot & Ext. Classpath   = 
  Application Classpath   = C:\Program Files\NetBeans-12.6\netbeans\platform\lib\boot.jar;C:\Program Files\NetBeans-12.6\netbeans\platform\lib\org-openide-modules.jar;C:\Program Files\NetBeans-12.6\netbeans\platform\lib\org-openide-util-lookup.jar;C:\Program Files\NetBeans-12.6\netbeans\platform\lib\org-openide-util-ui.jar;C:\Program Files\NetBeans-12.6\netbeans\platform\lib\org-openide-util.jar
  Startup Classpath       = C:\Program Files\NetBeans-12.6\netbeans\platform\core\asm-9.2.jar;C:\Program Files\NetBeans-12.6\netbeans\platform\core\asm-commons-9.2.jar;C:\Program Files\NetBeans-12.6\netbeans\platform\core\asm-tree-9.2.jar;C:\Program Files\NetBeans-12.6\netbeans\platform\core\core-base.jar;C:\Program Files\NetBeans-12.6\netbeans\platform\core\core.jar;C:\Program Files\NetBeans-12.6\netbeans\platform\core\org-netbeans-libs-asm.jar;C:\Program Files\NetBeans-12.6\netbeans\platform\core\org-openide-filesystems-compat8.jar;C:\Program Files\NetBeans-12.6\netbeans\platform\core\org-openide-filesystems.jar;C:\Program Files\NetBeans-12.6\netbeans\nb\core\org-netbeans-upgrader.jar;C:\Program Files\NetBeans-12.6\netbeans\nb\core\locale\core_nb.jar
-------------------------------------------------------------------------------
WARNING [org.netbeans.JarClassLoader]: Opening C:\Program Files\NetBeans-12.6\netbeans\ide\modules\ext\servlet-2.2.jar took 606 ms
WARNING [org.netbeans.JarClassLoader]: Opening C:\Program Files\NetBeans-12.6\netbeans\java\modules\ext\jaxws22\api\jaxws-api.jar took 876 ms
WARNING [org.netbeans.JarClassLoader]: Opening C:\Program Files\NetBeans-12.6\netbeans\java\modules\ext\jaxws22\ha-api.jar took 559 ms
WARNING [org.netbeans.JarClassLoader]: Opening C:\Program Files\NetBeans-12.6\netbeans\java\modules\ext\jaxws22\mimepull.jar took 567 ms
WARNING [org.netbeans.JarClassLoader]: Opening C:\Program Files\NetBeans-12.6\netbeans\java\modules\ext\jaxws22\management-api.jar took 774 ms
WARNING [org.netbeans.JarClassLoader]: Opening C:\Program Files\NetBeans-12.6\netbeans\platform\modules\ext\hamcrest-core-1.3.jar took 622 ms
INFO [org.netbeans.modules.netbinox]: Install area set to file:/C:/Program Files/NetBeans-12.6/netbeans/
WARNING [org.netbeans.core.modules]: the modules [org.netbeans.modules.xml.text] use org.netbeans.modules.editor.deprecated.pre65formatting which is deprecated.
WARNING [org.netbeans.core.modules]: the modules [org.netbeans.modules.ide.kit, org.netbeans.modules.xml.text] use org.netbeans.modules.editor.structure which is deprecated.
WARNING [org.netbeans.core.modules]: the modules [org.netbeans.modules.ant.hints, org.netbeans.modules.apisupport.ant, org.netbeans.modules.java.hints, org.netbeans.modules.jshell.support, org.netbeans.modules.maven.hints] use org.netbeans.modules.java.hints.legacy.spi which is deprecated: Use Java Hints SPI (org.netbeans.spi.java.hints) instead.

### Did this work correctly in an earlier version?

No

### Operating System

Windows 11

### JDK

15.0.1

### Apache NetBeans packaging

Apache NetBeans provided installer

### Anything else

It notifies me of slowness detected

### Are you willing to submit a pull request?

No

### Code of Conduct

Yes

**Comment by geertjanw** [comment]
You're using JDK 15.0.1?

Probably better to use an LTS release?

**Comment by neilcsmith-net** [comment]
As there are no reproduction steps here I'm going to convert over to discussions - it's unclear what the actual issue (aside from the slowness report) is. The warnings shared don't seem to reference JavaFX?!
